src/components/fields/edit/SelectionEditor.jsx

```diff
@@ -23,19 +23,26 @@ const SelectionEditor = function(props) {
     ...rest
   } = props;
   const intl = useIntl();
-  const splitValues =
-    typeof value === 'string' ? value?.split(',') : value;
+  const isMultiselect = schema.fieldType === fieldTypes.multiselect;
+  const isAlreadyArray = Array.isArray(value);
+  const makeSafeArray = val => {
```

Comment:
I imagine that there's a cleaner way to do this, but I couldn't immediately think of it.